code/datums/outfits/outfit.dm

```diff
@@ -39,6 +40,8 @@
 	var/list/accessory_contents = list()
 	var/list/belt_contents = list() //In the list(path=count,otherpath=count) format
 	var/list/implants = null //A list of implants that should be implanted
+	var/spell_level
+	var/list/spells = list() // A list of spells to grant
```

Comment:
I feel like this could just be an assoc list where `spells[spell_path]` gives you the spell level for that spell